web/libs/editor/src/components/SidePanels/TabPanels/Tabs.tsx

```diff
@@ -229,9 +238,38 @@ export const Tabs = (props: BaseProps) => {
             );
           })}
           <Elem id={`${props.name}_${props.panelViews.length}-droppable-space`} name="drop-space-after" />
+          {props.isBottomPanel && props.settings?.collapsibleBottomPanel && (
+            <Button
+              className="collapsible-bottom-panel-toggle"
+              // TODO: remove inline styles and use tailwind classes when Button component is updated
+              style={{
+                marginLeft: 4,
+                marginRight: 5,
+                marginTop: 4,
+                display: "flex",
+                height: "24px",
+                width: "24px",
+                padding: 0,
+                alignItems: "center",
+                background: "none",
+                border: "none",
+                cursor: "pointer",
+              }}
```

Comment:
agreed with the todo comment, can we get that going now? or should we wait for a follow up?

Reply:
I didn't want to touch the button components at all, so we change it to proper Tailwind classes as soon as the new Button PR lands.